public/app/features/alerting/unified/components/amroutes/AmSpecificRouting.tsx

```diff
@@ -108,12 +168,32 @@ export const AmSpecificRouting: FC<AmSpecificRoutingProps> = ({
 const getStyles = (theme: GrafanaTheme2) => {
   return {
     container: css`
+      display: flex;
+      flex-flow: column wrap;
+    `,
+    searchContainer: css`
+      display: flex;
+      flex-flow: row nowrap;
+      padding-bottom: ${theme.spacing(2)};
+      border-bottom: 1px solid ${theme.colors.border.strong};
+    `,
+    clearFilterBtn: css`
+      align-self: flex-end;
+      margin-left: ${theme.spacing(1)};
+    `,
+    filterInput: css`
+      width: 340px;
+      & + & {
```

Comment:
Interesting selector, I might use this one in the future :)

Reply:
I've noticed it when working on this feature :) Might be quite useful in some cases